UPGRADING.md

```diff
@@ -112,6 +112,34 @@ The following inputs are no longer available:
 The inputs were marked as deprecated since Graylog version `3.2.0`.
 If you still run any of those inputs, please configure the alternative "Kinesis/CloudWatch" input instead ahead of upgrading.
 
+## CrowdStrike input log parsing changes
+
+Several log parsing changes have been made to the CrowdStrike input.
+
+Added fields:
+`event_created`: Contains the `metadata.eventCreationTime` log value.
+`vendor_subtype`: Contains the `metadata.eventType` log value.
+`vendor_version`: Contains the `metadata.version` log value.
+`event_source_product`: Contains the static value `crowdstrike_falcon`.
+
+Changed fields:
+- `message`: Now contains the JSON content of the log `event` value, effectively the message payload.
+- The message `timestamp` field is now set to the current Graylog system date/time, instead of the previously used log `metadata.eventCreationTime` value.
+
+Removed fields:
+- `event_end`
+- `event_source`
+- `event_start`
+- `user_domain`
+- `user_id`
+- `vendor_event_description`
+- `FILE_NAME`
+- `FILE_PATH`
+- `OBJECTIVE`
+- `TECHNIQUE`
+
+Note that additional CrowdStrike message parsing is expected to be released in a future release of Graylog Illuminate.
+
```

Comment:
I don't know if we usually backport changes to the UPGRADING.md. I thought it was supposed to only be for changes when upgrading to the new major/minor version, but in this case 5.1.0-5.1.5 will not have the changes listed here. Not saying it's right or wrong to backport this, I've just not seen it before.

Reply:
I agree, this scenario is definitely not that common, and this file usually makes the most sense for major versions and the first minor version. But, since we are introducing breaking log parsing changes in a minor upgrade, I think we should give as much heads up to users as possible and include the changes in this file. I agree that users going from 5.1.5 to 5.1.6 are less likely to see it, but there is also the chance of a user going to 5.1.6 from an older major version, and they should see those changes.

Reply:
Oops, sorry, I forgot that CrowdStrike was introduced in 5.1, so the older minor version upgrade would not apply. But, I think the changes are good to include, just in case a user does review the file.

Reply:
@kingzacko1 @ryan-carroll-graylog What do you think about making a note in the new section to indicate that the changes were introduced in 5.1.6 to help with clarity?

Reply:
Seems solid to me and better to err on the side of more info. At the very least it definitely can't hurt.

Reply:
I agree. 99% of the time it probably won't make any difference at all, but I think at least mentioning that these changes aren't present until 5.1.6 is worth it.

Reply:
I agree, probably very slim chance it's used (especially given how new the CrowdStrike input is, and given the bugs that the input had with it - probably not many folks it at all using it). I went ahead and added the version note. Let me know if that looks good to ya'll.